src/http/server/handlers/deflate_handler.cr

```diff
@@ -0,0 +1,33 @@
+require "zlib"
+
+class HTTP::DeflateHandler < HTTP::Handler
+  DEFLATE_TYPES = %w(text/html text/plain text/xml text/css text/javascript application/javascript)
```

Comment:
It would be nice to be able to configure this list. It misses `application/json` too.